xml/containers-basics.xml

```diff
@@ -56,10 +66,16 @@
   </listitem>
   <listitem>
    <para>
-    Containers are not secure, and the overall security depends on the host
-    system. Containerized applications can be secured through &aa; or &selnx;
-    profiles. Securing containers is harder than securing virtual machines, due
-    to the larger attack surface.
+     A given container will encapsulate binaries for a given architecture (x86_64 or aarch64 for 
+     instance). So a container made for x86_64 will only run on a x86_64 system host.
```

Comment:
This is not really a limitation of containers though: VMs are also build for a specific architecture and you usually do not want to run them on foreign architectures.

Reply:
Agreed, I was trying "hard" to find some advantage and since "portability" is written everywhere for containers, I wanted to state this "obvious".